### ISSUE TITLE
All copy from language buttons invisible except last one

### ISSUE THREAD

**Issue by stuaxo** [infoportugal/wagtail-modeltranslation#202]
I've got a few languages on my site, the last one being zh-cn only this button is visible.

This is because the rest have somehow moved down in their container in the dom. 

Untoggling the rule for .translation-field-copy-wrapper > button width makes it visible again - video:

https://youtu.be/FHn34_7y1M0

**Comment by stuaxo** [comment]
If anyone needs a workaround, here is mine using a global hook to add css to the admin:

put wagtail_books.py in one of your apps:
```python
from django.utils.html import format_html
from django.contrib.staticfiles.templatetags.staticfiles import static

from wagtail.core import hooks


@hooks.register('insert_global_admin_css')
def global_admin_css():
    return format_html('<link rel="stylesheet" href="{}">', static('css/admin-extras.css'))
```

admin-extras css in my static files / css:

```css
.translation-field-copy-wrapper {
    margin-top: -6px;
}

.translation-field-copy-wrapper .translation-field-copy {
    width: auto;
    margin-top: -6px;
}
```

width makes the buttons visible, the margin-top, stops the bottom of the buttons being cut off.

**Comment by dmarcelino** [comment]
Hi @stuaxo, thanks for the workaround!

Any chance you can submit a PR with a proper fix?

Thanks

**Comment by stuaxo** [comment]
I can, but I'd like a bit more info on what a proper fix is.

I'm guess removing the width from the original css, and putting the -6px hack in should be enough ?

**Comment by dmarcelino** [comment]
Cool.

I honestly don't know as I haven't looked at that part of `wagtail-modeltranslation` at all. But perhaps @DiogoMarques29 can help you.

**Comment by stuaxo** [comment]
Cheers, I'll try and search github for the relevant css in github, any guidance definitely welcome.

**Comment by DiogoMarques29** [comment]
Fixed here: 3ec4d10875eb9a8326a5da4c9e761b4fde4f99e1
